### DISCUSSION TITLE
can't save data using grafana cloud

### DISCUSSION THREAD

**Discussion by fakih1993** [grafana/grafana-infinity-datasource#399] [Q&A]
I using grafana cloud, already get data from IOT platform , but it is only can subscribe latest data 
how  to save this data  ?  because i need to make a graphic
![image](https://user-images.githubusercontent.com/95876376/189560781-54c240ca-4d77-4f82-b2fa-515e44b2fec9.png)

this is my API setting
![image](https://user-images.githubusercontent.com/95876376/189561183-59b702d6-6a59-4f09-a910-08717922a045.png)

**Comment by yesoreyeram** [comment]
The plugin just do visualisation part. It can help here much. Either you need some sort of Prometheus exporters or enterprise recorded queries feature
